struqture/src/spins/pauli_product.rs

```diff
@@ -674,7 +679,7 @@ impl FromStr for PauliProduct {
                                 "Using {} instead of unsigned integer as spin index",
                                 index
                             ),
-                        })
+                        });
```

Comment:
How come this has been added to this file and not the others?

Reply:
I modified this file when I was testing to find where the error was coming from I must have modified this at that time. Since its at the end of a return statement the `;` does nothing so I didn't notice.